index.js

```diff
@@ -2,91 +2,119 @@
 
 'use strict';
 
-var tls = require('tls');
-var net = require('net');
-var eos = require('end-of-stream');
-var through = require('through2');
-var minimist = require('minimist');
-var allContainers = require('docker-allcontainers');
-var statsFactory = require('docker-stats');
-var logFactory = require('docker-loghose');
-var eventsFactory = require('docker-event-log');
-var os = require('os');
+const tls = require('tls');
+const net = require('net');
+const eos = require('end-of-stream');
+const through = require('through2');
+const minimist = require('minimist');
+const allContainers = require('docker-allcontainers');
+const statsFactory = require('docker-stats');
+const logFactory = require('docker-loghose');
+const eventsFactory = require('docker-event-log');
+const os = require('os');
+
+let debugLogging = !!process.env.INSIGHT_DOCKER_DEBUG;
```

Comment:
AFAIK double negation is bad practice, would prefer this
```suggestion
let debugLogging = Boolean(process.env.INSIGHT_DOCKER_DEBUG);
```

Reply:
no difference

Reply:
more readable